src/words/WordsToReview.js

```diff
@@ -11,39 +18,178 @@ export default function WordsToReview({
   notifyWordChanged,
   source,
 }) {
+  const totalWordsTranslated = words.length;
+  const [inEditMode, setInEditMode] = useState(false);
+  const [wordsForExercises, setWordsForExercises] = useState([]);
+  const [wordsExcludedForExercises, setWordsExcludedForExercises] = useState(
+    [],
+  );
+  const [totalWordsSelectedByZeeguu, setTotalWordsSelectedByZeeguu] =
+    useState();
+  const [totalWordsEditedByUser, setTotalWordsEditedByUser] = useState();
+  const [wordsExpressions, setWordsExpressions] = useState([]);
+  const [showExplainWordSelectionModal, setShowExplainWordSelectionModal] =
+    useState(false);
+
+  useEffect(() => {
+    let newWordsForExercises = [];
```

Comment:
I still think I would prefer if the local temporary variables would be named with `_` in front of them instead of `new`. I'd be happy to join the search party (search-and-replace party) for all of them around the codebase :)

Reply:
I think if you believe it will make more sense in the long run we can do that, we should do it - but maybe let's take this as it is and then do a replace that's just renaming?